slack-api-model/src/main/java/com/slack/api/model/Attachment.java

```diff
@@ -357,6 +357,7 @@ public static class Record {
     private String mimetype;
     private String url;
     private AttachmentMetadata metadata;
+    private Object workObjectEntity;
```

Comment:
👾 question: Is this particular to a specific `Object` import, or can it be cast to something else? IIRC adjustments to these models are common if fields are adjusted, but having such type might make accessing easier?

Reply:
Removed this property since we're removing it from API responses and events